apps/analog-app/src/app/pages/shipping/index.page.ts

```diff
@@ -1,22 +1,31 @@
-import { AsyncPipe, CurrencyPipe, NgForOf } from '@angular/common';
-import { Component, inject, OnInit } from '@angular/core';
+import { AsyncPipe, CurrencyPipe } from '@angular/common';
+import { Component, inject } from '@angular/core';
 
 import { Observable } from 'rxjs';
 import { CartService } from '../../cart.service';
 
 @Component({
   selector: 'app-shipping',
   standalone: true,
-  imports: [NgForOf, CurrencyPipe, AsyncPipe],
-  templateUrl: './shipping.html',
-  styleUrls: ['./shipping.scss'],
+  imports: [CurrencyPipe, AsyncPipe],
+  template: `
+    <h3>Shipping Prices</h3>
+    @for (shipping of shippingCosts | async; track $index) {
+    <div class="shipping-item">
+      <span>{{ shipping.type }}</span>
+      <span>{{ shipping.price | currency }}</span>
+    </div>
+    }
+  `,
+  styles: `
+  h3 {
+  border: solid 1px darkblue;
+  }
+  `,
 })
-export default class ShippingComponent implements OnInit {
-  private cartService = inject(CartService);
-
-  shippingCosts!: Observable<{ type: string; price: number }[]>;
+export default class ShippingComponent {
+  private readonly cartService = inject(CartService);
 
-  ngOnInit(): void {
-    this.shippingCosts = this.cartService.getShippingPrices();
-  }
+  shippingCosts: Observable<{ type: string; price: number }[]> =
```

Comment:
minor: Type can be inferred now since we are directly assigning it. This was needed previously because the assignment happened separately from declaration.
```suggestion
  shippingCosts =
```
Also, this line will need Prettier to be run again.

Reply:
I've removed my comments regarding prettier and made those changes in !1604 - No impact on your PR here!